Doc/library/ipaddress.rst

```diff
@@ -117,6 +116,12 @@ write code that handles both IP versions correctly.  Address objects are
    >>> ipaddress.IPv4Address(b'\xC0\xA8\x00\x01')
    IPv4Address('192.168.0.1')
 
+   .. versionchanged:: 3.8
+
+      Leading zeros are no longer tolerated and are treated as an error.
+      IPv4 address strings are now parsed as strict as glibc
+      :func:`~socket.inet_pton`.
+
```

Comment:
This is incorrect. You should use the wording from GH-25099 for `.. versionchanged:: 3.8` *and* then add `.. versionchanged:: 3.8.12` with the content you have here.

Reply:
:+1: